lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -60,6 +61,28 @@ function getNodesAndTimingByUrl(nodeTimings) {
   return urlMap;
 }
 
+/**
+ * @param {Node} node
+ * @param {LH.Gatherer.Simulation.NodeTiming} nodeTiming
+ * @param {LH.Artifacts.DetectedStack[]} Stacks
+ */
+function computeStackSpecificTiming(node, nodeTiming, Stacks) {
+  const stackSpecificTiming = Object.assign({}, nodeTiming);
+  if (!Stacks) return stackSpecificTiming;
+  if (Stacks.some(stack => stack.id === 'amp')) {
+    // AMP will load a linked stylesheet asynchronously if it has not been loaded after 1 second [https://github.com/ampproject/amphtml/blob/master/src/font-stylesheet-timeout.js]
```

Comment:
protip: press `y` while on a github blob to turn the shortcut ref into a permalink of the current hash, this way if amp ever renames its master branch to main or moves the file future readers will still be able to find what you were talking about :)
```suggestion
    // AMP will load a linked stylesheet asynchronously if it has not been loaded after 1 second
    // https://github.com/ampproject/amphtml/blob/8e03ac2f315774070651584a7e046ff24212c9b1/src/font-stylesheet-timeout.js#L54-L59
```